pages/enterprise/changelog.rst

```diff
@@ -2,6 +2,63 @@
 Changelog
 *********
 
+Graylog Enterprise 2.2.3
+========================
+
+Released: 2017-04-04
+
+Plugin: Archive
+---------------
+
+* Metadata is now stored in MongoDB
+* Preparation for storage backend support
+
+Graylog Enterprise 2.2.2
+========================
+
+Released: 2017-03-02
+
+Plugin: Audit Log
+-----------------
+
+Extend intergation with the Archive plugin
+
+Graylog Enterprise 2.2.1
+========================
+
+Released: 2017-02-20
+
+Plugin: Archive
+---------------
+
+Improve stability and smaller UI fixes
+
+Graylog Enterprise 2.2.0
+========================
+
+Released: 2017-02-09
+
+Plugin: Archive
+---------------
+
+Improve index set support
+
+Graylog Enterprise 1.2.1
+========================
+
+Released: 2017-01-26
+
+Plugin: Archive
+---------------
+
+Prepare for the plugin to be compatible with the new default stream.
+
+Plugin: Audit Log
+-----------------
+
+Add support for index sets and fix potential NPEs.
+Smaller UI imprevements.
```

Comment:
In the end, all change items should be in a bullet list, for a consolidated look.